vm/src/stdlib/codecs.rs

```diff
@@ -238,6 +315,16 @@ mod _codecs {
         do_codec!(utf8::decode, args, vm)
     }
 
+    #[pyfunction]
+    fn ascii_encode(args: EncodeArgs, vm: &VirtualMachine) -> EncodeResult {
+        do_codec!(ascii::encode, args, vm)
```

Comment:
Does special handling for ascii string helpful to avoid char enumeration?
```suggestion
        if args.s.is_ascii() {
            return Ok(args.s.to_bytes());
        }
        do_codec!(ascii::encode, args, vm)
```